Update changelog server.md RU for 24.1

katidisergei opened the pull request:
### Changelog entry <!-- a user-readable short description of changes introduced in this PR -->

Update changelog server.md RU for 24.1

### Changelog category <!-- remove all except one -->

* Documentation (changelog entry is not required)

### Additional information

...

katidisergei committed:
Update changelog-server.md for 24.1 RU 

Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md with EN descriptions  of the 23-4 server version

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

katidisergei committed:
Update changelog-server.md for 24.1 RU

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+**Производительность:**
+
+* [Улучшена производительность](https://github.com/ydb-platform/ydb/pull/1576) в запросах возвращающих однострочные ответы.
```
Поскольку мы заявляем появление QueryService, оптимизации производительности в нём можно не упоминать

katidisergei replied:
убрал

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+**Производительность:**
+
+* [Улучшена производительность](https://github.com/ydb-platform/ydb/pull/1576) в запросах возвращающих однострочные ответы.
+* [Оптимизированы](https://github.com/ydb-platform/ydb/pull/1837) таймауты сессии от сервера до клиента, ранее таймаут составлял 5 секунд, что в худшем случае приводило к определению неработающего клиента в течении 10 секунд, время проверки теперь зависит от времени ожидания сеанса, что приводит к более быстрой смене лидера.
```
```suggestion
* [Оптимизированы](https://github.com/ydb-platform/ydb/pull/1837) таймауты сессий сервиса координации от сервера до клиента. Ранее таймаут составлял 5 секунд, что в худшем случае приводило к определению неработающего клиента (и освобождению удерживаемых им ресурсов) в течение 10 секунд. В новой версии время проверки зависит от времени ожидания сеанса, что приводит к более быстрому реагированию при смене лидера или захвате распределенных блокировок.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+* [Улучшена производительность](https://github.com/ydb-platform/ydb/pull/1576) в запросах возвращающих однострочные ответы.
+* [Оптимизированы](https://github.com/ydb-platform/ydb/pull/1837) таймауты сессии от сервера до клиента, ранее таймаут составлял 5 секунд, что в худшем случае приводило к определению неработающего клиента в течении 10 секунд, время проверки теперь зависит от времени ожидания сеанса, что приводит к более быстрой смене лидера.
+* [Оптимизировано](https://github.com/ydb-platform/ydb/pull/2391) потребление CPU schemeshard репликами особенно при обработке быстрых обновлений для таблиц с большим количеством разделов. 
```
```suggestion
* [Оптимизировано](https://github.com/ydb-platform/ydb/pull/2391) потребление CPU SchemeShard репликами, особенно при обработке быстрых обновлений для таблиц с большим количеством разделов. 
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+**Исправления ошибок:**
+
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4105) ошибка приводящая к нарушению порядка передаваемых изменений при асинхронной репликации данных. 
```
Выше мы заявляем асинхронную репликацию как новую функцию. Можно исключить.

katidisergei replied:
убрал

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4105) ошибка приводящая к нарушению порядка передаваемых изменений при асинхронной репликации данных. 
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, CDC резервирует емкость очереди изменений при первоначальном сканировании.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между запросом записей и их отправкой.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между получением записей CDC и их отправкой.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4105) ошибка приводящая к нарушению порядка передаваемых изменений при асинхронной репликации данных. 
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, CDC резервирует емкость очереди изменений при первоначальном сканировании.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между запросом записей и их отправкой.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
```
Ошибка специфична для 24-1, исправлена для 24-1 же. Можно не упоминать в заметках к выпуску версии 24-1

katidisergei replied:
пока оставлю для количества )

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между запросом записей и их отправкой.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2305) проблема, когда shared ноды не возвращались из backend для отображения в Embedded UI для serverless баз.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) проблема, когда при включенных и используемых волатильных транзакциях, возвращался успешный результат коммита транзакции которая на самом деле завершилась неудачей.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) редко возникающая ошибка, когда при включенных и используемых волатильных транзакциях возвращался успешный результат подтверждения транзакции, которая на самом деле завершилась неудачей.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2305) проблема, когда shared ноды не возвращались из backend для отображения в Embedded UI для serverless баз.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) проблема, когда при включенных и используемых волатильных транзакциях, возвращался успешный результат коммита транзакции которая на самом деле завершилась неудачей.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/2839) проблема с блокировками в итераторе чтения при игнорировании пустых результатов.
```
```suggestion
* [Устранена](https://github.com/ydb-platform/ydb/pull/2839) редко возникающая ошибка, приводившая к потере установленных блокировок и успешному подтверждению транзакций, которые должны были завершиться ошибкой TLI.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2305) проблема, когда shared ноды не возвращались из backend для отображения в Embedded UI для serverless баз.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) проблема, когда при включенных и используемых волатильных транзакциях, возвращался успешный результат коммита транзакции которая на самом деле завершилась неудачей.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/2839) проблема с блокировками в итераторе чтения при игнорировании пустых результатов.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) проблема когда немедленное (но локально обновлённое до повторяемого) чтение вызывало путаницу зависимостей между одновременными записями, что позволяло зафиксировать записи для одного и того же ключа в обратном порядке версий MVCC.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) редкая ошибка, приводящая к возможному нарушению гарантий целостности данных при конкурентном записи и чтении данных по определенному ключу.
```

blinkov replied:
про гарантии целостности страшновато звучит — хорошо бы более конкретно сформулировать как оно выглядело с точки зрения пользователя и при каких обстоятельствах

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Устранена](https://github.com/ydb-platform/ydb/pull/2839) проблема с блокировками в итераторе чтения при игнорировании пустых результатов.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) проблема когда немедленное (но локально обновлённое до повторяемого) чтение вызывало путаницу зависимостей между одновременными записями, что позволяло зафиксировать записи для одного и того же ключа в обратном порядке версий MVCC.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4343) проблема, когда реплики на чтение переставали обрабатывать запросы.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4979) ошибка когда после переименования таблицы могли остаться незафиксированные изменения. В худшем случае постоянные блокировки могли восстановиться после перезапуска сегмента, что привело бы к нарушению инварианта при попытке сохранения.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/4979) редкая ошибка, которая могла привести к аварийному завершению процессов базы данных при наличии неподтвержденных транзакций над таблицей в момент ее переименования.
```

zinal commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) проблема когда немедленное (но локально обновлённое до повторяемого) чтение вызывало путаницу зависимостей между одновременными записями, что позволяло зафиксировать записи для одного и того же ключа в обратном порядке версий MVCC.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4343) проблема, когда реплики на чтение переставали обрабатывать запросы.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4979) ошибка когда после переименования таблицы могли остаться незафиксированные изменения. В худшем случае постоянные блокировки могли восстановиться после перезапуска сегмента, что привело бы к нарушению инварианта при попытке сохранения.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/5081) ошибка когда запрос на обновление таблицы не менял значения в таблице, хотя операция помечена как успешная.
```
Ошибка специфична для 24-1 и выше. Предлагаю исключить из заметок к выпуску по 24-1.

katidisergei replied:
Все 3 убрать или только https://github.com/ydb-platform/ydb/pull/5081 ?

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+**Исправления ошибок:**
+
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4105) ошибка приводящая к нарушению порядка передаваемых изменений при асинхронной репликации данных. 
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, CDC резервирует емкость очереди изменений при первоначальном сканировании.
```
дать ссылку на что такое cdc

katidisergei replied:
добавил

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, CDC резервирует емкость очереди изменений при первоначальном сканировании.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между запросом записей и их отправкой.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2305) проблема, когда shared ноды не возвращались из backend для отображения в Embedded UI для serverless баз.
```
serverless — облачная специфика, тут ни к чему

katidisergei replied:
убрал

katidisergei commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+Дата выхода: 31 июля 2024.
+
+**Функциональность:**
```
Добавил текстом, но описания не смог найти в документации, будет круто если подкинете ссылки

katidisergei commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+**Функциональность:**
+
+* [Реализована](https://ydb.tech/docs/ru/concepts/async-replication) асинхронная репликация между базами данных YDB https://github.com/ydb-platform/ydb/pull/5418
```
Поправил существующие две

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,40 @@
+
+**Встроенный UI:**
+
+* Добавлен дашборд диагностики потребления ресурсов.
+* Добавлены графики для мониторинга основных показателей работы кластера YDB.
```
уточняю

katidisergei replied:
подробностей нет

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Maksim Zinal <MaxZinal@yandex.ru>

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

попоравил YDB на {{ ydb-short-name }}

katidisergei committed:
Update changelog-server.md

добавил инфо про LDAP и описание нескольких фиксов

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

убрал "Ошибка специфична для 24-1 и выше. Предлагаю исключить из заметок к выпуску по 24-1.
"

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, [Change Data Capture](./dev/cdc.md) резервирует емкость очереди изменений при первоначальном сканировании.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между получением записей CDC и их отправкой.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) редко возникающая ошибка, когда при включенных и используемых волатильных транзакциях возвращался успешный результат подтверждения транзакции, которая на самом деле завершилась неудачей.
+* [Устранена](https://github.com/ydb-platform/ydb/pull/2839) редко возникающая ошибка, приводившая к потере установленных блокировок и успешному подтверждению транзакций, которые должны были завершиться ошибкой TLI.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) редкая ошибка, приводящая к возможному нарушению гарантий целостности данных при конкурентном записи и чтении данных по определенному ключу.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4343) проблема, когда реплики на чтение переставали обрабатывать запросы.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4979) редкая ошибка, которая могла привести к аварийному завершению процессов базы данных при наличии неподтвержденных транзакций над таблицей в момент ее переименования.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3632) ошибка в логике вычисления статуса  статической группы, когда статическая группа не помечалась нерабочей, хотя должна была.
+* [Исправлено](https://github.com/ydb-platform/ydb/pull/1835) аварийное завершение отладочной сборки и неправильный тип результирующего сообщения, в случае остановки таблеток даташарда во время работы некоторых операций.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичного коммита распределённой транзакции с незакомиченными изменениями в случае некоторых гонок с рестартами.
+* [Исправлены](https://github.com/ydb-platform/ydb/pull/2374) аномалии с чтением устаревших данных, которые были обнаружены с помощью Jepsen.
```
```suggestion
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) редко возникающая ошибка, когда при включённых и используемых волатильных транзакциях возвращался успешный результат подтверждения транзакции до того, как она была успешно закоммичена.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2839) редко возникающая ошибка, приводившая к потере установленных блокировок и успешному подтверждению транзакций, которые должны были завершиться ошибкой Transaction Locks Invalidated.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3074) редкая ошибка, приводящая к возможному нарушению гарантий целостности данных при конкурентной записи и чтении данных по определённому ключу.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/4343) проблема, из-за которой реплики для чтения переставали обрабатывать запросы.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/4979) редкая ошибка, которая могла привести к аварийному завершению процессов базы данных при наличии неподтверждённых транзакций над таблицей в момент её переименования.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/3632) ошибка в логике определения статуса статической группы, когда статическая группа не помечалась нерабочей, хотя должна была.
* [Исправлена](https://github.com/ydb-platform/ydb/pull/2169) ошибка частичного коммита распределённой транзакции с незакоммиченными изменениями в случае некоторых гонок с рестартами.
* [Исправлены](https://github.com/ydb-platform/ydb/pull/2374) аномалии с чтением устаревших данных, которые были [обнаружены с помощью Jepsen](https://blog.ydb.tech/hardening-ydb-with-jepsen-lessons-learned-e3238a7ef4f2).
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+* [Реализована](https://github.com/ydb-platform/ydb/pull/5418) [асинхронная репликация](./concepts/async-replication.md) между базами данных {{ ydb-short-name }}. 
+* Реализована [Knn UDF](./yql/reference/yql-core/udf/list/knn.md) для точного поиска ближайших векторов.
+* Разработан сервис QueryService, обеспечивающий возможность выполнения всех типов запросов (DML, DDL) и выборку неограниченных объёмов данных.
+* Реализована поддержка [аутентификации](./concepts/auth.md) и получения перечня групп из внешних LDAP-каталогов.
```
звучит как-будто аутентификации раньше совсем не было, что явно не так — о чём именно речь?

katidisergei replied:
Поправил формулировку немного

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+
+* [Реализована](https://github.com/ydb-platform/ydb/pull/5418) [асинхронная репликация](./concepts/async-replication.md) между базами данных {{ ydb-short-name }}. 
+* Реализована [Knn UDF](./yql/reference/yql-core/udf/list/knn.md) для точного поиска ближайших векторов.
+* Разработан сервис QueryService, обеспечивающий возможность выполнения всех типов запросов (DML, DDL) и выборку неограниченных объёмов данных.
```
```suggestion
* Разработан gRPC сервис QueryService, обеспечивающий возможность выполнения всех типов запросов (DML, DDL) и выборку неограниченных объёмов данных.
```

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+* [Реализована](https://github.com/ydb-platform/ydb/pull/5418) [асинхронная репликация](./concepts/async-replication.md) между базами данных {{ ydb-short-name }}. 
+* Реализована [Knn UDF](./yql/reference/yql-core/udf/list/knn.md) для точного поиска ближайших векторов.
+* Разработан сервис QueryService, обеспечивающий возможность выполнения всех типов запросов (DML, DDL) и выборку неограниченных объёмов данных.
+* Реализована поддержка [аутентификации](./concepts/auth.md) и получения перечня групп из внешних LDAP-каталогов.
```
дока про группы из ldap https://ydb.tech/docs/ru/concepts/auth#poluchenie-grupp

katidisergei replied:
Если так добавить, то не на .md  будет заканчиваться + в этой строке есть ссылка на доку где как раз содержится инфа про группы.

blinkov commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+
+**Встроенный UI:**
+
+* Добавлен дашборд диагностики потребления ресурсов.
+* Добавлены графики для мониторинга основных показателей работы кластера {{ ydb-short-name }}.
```
см. в telegram

katidisergei replied:
посмотрел, ниже комент Антона, дописал.

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei committed:
Update ydb/docs/ru/core/changelog-server.md

Co-authored-by: Ivan Blinkov <ivan@blinkov.ru>

katidisergei commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,42 @@
+
+**Встроенный UI:**
+
+* Добавлен дашборд диагностики потребления ресурсов.
```
Дописал

katidisergei committed:
Update changelog-server.md

fomichev3000 commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,41 @@
+
+**Функциональность:**
+
+* [Реализована](https://github.com/ydb-platform/ydb/pull/5418) [асинхронная репликация](./concepts/async-replication.md) между базами данных {{ ydb-short-name }}. 
```
мне кажется, что в 24-1 она нифига не реализвована. Я бы удалил

katidisergei replied:
Убрал. AR появилась  в 24-1-async-replication, затем потребовалось несколько исправлений, анонсировать буду в след версиях.

fomichev3000 commented on ydb/docs/ru/core/changelog-server.md:
```diff
@@ -1,5 +1,41 @@
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/3917) ошибка возможного переполнения очереди, [Change Data Capture](./dev/cdc.md) резервирует емкость очереди изменений при первоначальном сканировании.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/4597) потенциальная взаимоблокировка между получением записей CDC и их отправкой.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2056) прблема потери очереди задач медиатора при переподключении медиатора, исправление позволяет обработать очередь задач медиатора при ресинхронизации.
+* [Исправлена](https://github.com/ydb-platform/ydb/pull/2624) редко возникающая ошибка, когда при включённых и используемых волатильных транзакциях возвращался успешный результат подтверждения транзакции до того, как она была успешно закоммичена.
```
Можно добавить, что волатильные транзакции выключены по умолчанию, находятся в разработке

katidisergei replied:
Поправил

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md

no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]

katidisergei committed:
Update changelog-server.md

fix links

katidisergei committed:
Update changelog-server.md

katidisergei committed:
Update changelog-server.md
